capella/setup/disable_semantic_browser_auto_refresh.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+import fileinput
+import logging
+import os
+import pathlib
+import sys
+
+from lxml import etree
+
+NS_MAP = {
+    "xmi": "http://www.omg.org/XMI",
+    "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+    "basic": "http://www.eclipse.org/ui/2010/UIModel/application/ui/basic",
+    "advanced": (
+        "http://www.eclipse.org/ui/2010/UIModel/application/ui/advanced"
+    ),
+    "application": "http://www.eclipse.org/ui/2010/UIModel/application",
+    "menu": "http://www.eclipse.org/ui/2010/UIModel/application/ui/menu",
+}
+XPATH_EXPR = "//persistedState[@key='memento' and contains(@value, 'listeningToWorkbenchPageSelectionEvents')]"
+WORKSPACE_DIR = os.getenv("WORKSPACE_DIR", "/workspace")
+
+logging.basicConfig(level=os.getenv("LOG_LEVEL", "INFO"))
+logger = logging.getLogger(__file__)
+
+
+def check_environment_variable() -> None:
+    disable_semantic_browser = os.getenv(
+        "CAPELLA_DISABLE_SEMANTIC_BROWSER_AUTO_REFRESH", "0"
+    )
+    if disable_semantic_browser.lower() not in (
```

Comment:
```suggestion
    if disable_auto_refresh.lower() not in (
```
we disable the refresh and not the browser

Reply:
Ok, renamed to `disable_semantic_browser_auto_refresh` :)

Reply:
Fine. Something the module name tells us too. A German might say "Doppelt hält besser."